rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
+
```

Comment:
- [ ] Good job adding the getter and setter for the date instance variable. Please also set the Getters and Setters for `:book, and :person` instance variables.  This creates a lot of flexibility in how rental is created and accessed.

Reply:
Hi @TadesseDev ,

Thanks for your feedback and optional suggestion. I have updated the instance variables getter and setter.

Best Regards,
K. Inoulen Shanongba

Reply:
Good job @lenkon, As I have mentioned in the optional changes, Please make sure to have more commits in your project. Try to commit once you accomplish something and move on to the next one. this is a good practice and that is why `git` exists anyway.